src/components/buttons/Button.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import MUIButton from '@mui/material/Button';
+import { StyledEngineProvider } from '@mui/material';
+import css from './button.module.scss';
```

Comment:
```suggestion
import styles from './button.module.scss';
```
Lets use this naming, cause css is not really clear import variable naming